pkg/client/result.go

```diff
@@ -14,6 +16,31 @@ type Result struct {
 	Rows    []Row    `json:"rows"`
 }
 
+// Due to big int number limitations in javascript, numbers should be encoded
+// as strings so they could be properly loaded on the frontend.
+func (res *Result) PrepareBigints() {
+	for i, row := range res.Rows {
+		for j, col := range row {
+			if col == nil {
+				continue
+			}
+
+			switch reflect.TypeOf(col).Kind() {
+			case reflect.Int64:
+				val := col.(int64)
+				if val < -9007199254740991 || val > 9007199254740991 {
+					res.Rows[i][j] = strconv.FormatInt(col.(int64), 10)
+				}
```

Comment:
Pgweb is not considered to be used as a "real" api. So its api is exposed solely to be consumed by the frontend application.